sdk/src/androidTest/java/com/mongodb/stitch/StitchClientServiceTest.kt

```diff
@@ -0,0 +1,141 @@
+package com.mongodb.stitch
+
+import android.content.Context
```

Comment:
[convention] Should we split up 1st, 2nd, and 3rd party packages into their own lines?

Reply:
We'll have to set up the formatter to do this, though I'd like to see what Android Studio does by default.